src/main/flight/pid.c

```diff
@@ -529,8 +576,11 @@ rollAndPitchTrims_t *angleTrim, rxConfig_t *rxConfig)
     uint8_t axis;
     float ACCDeltaTimeINS, FLOATcycleTime, Mwii3msTimescale;
 
-//    MainDptCut = RCconstPI / (float)cfg.maincuthz;                           // Initialize Cut off frequencies for mainpid D
-    MainDptCut = RCconstPI / MAIN_CUT_HZ;                                      // maincuthz (default 12Hz, Range 1-50Hz), hardcoded for now
+    if (pidProfile->dterm_cut_hz > 0) {
+        MainDptCut = RCconstPI / pidProfile->dterm_cut_hz;                         // maincuthz (default 0 (disabled), Range 1-50Hz)
+    }
+    else { MainDptCut = 0;}
```

Comment:
the coding standard here would be:
```
if (pidProfile->dterm_cut_hz > 0) {
    MainDptCut = RCconstPI / pidProfile->dterm_cut_hz;                         // maincuthz (default 0 (disabled), Range 1-50Hz)
} else {
    MainDptCut = 0;
}
```

variable names are specifiedInCamelCase, defines and enums are all UPPER_CASE.